framework/components/ecl-timelines/ecl-timelines.config.js

```diff
@@ -1,13 +1,72 @@
 module.exports = {
   title: 'Timelines',
   label: 'Timelines',
-  status: 'planned',
-  collated: true,
+  status: 'ready',
   preview: '@preview-center-transparent',
   tags: ['molecule'],
   context: {
+    limit: 3,
+    button_label: 'Show all timeline',
+    items: [
+      {
+        title: 'By late-2013',
+        body: `
+          <p class="ecl-paragraph">
```

Comment:
Just pasting an idea from @emeryro (from the chat) https://twig.symfony.com/doc/2.x/tags/extends.html#block-nesting-and-scope :)